packages/create-nx-workspace/bin/create-nx-workspace.ts

```diff
@@ -120,6 +120,13 @@ const parsedArgs: any = yargsParser(process.argv.slice(2), {
   },
 }) as any;
 
+if (parsedArgs.version) {
```

Comment:
What is this supposed to do? :eyes:

Reply:
Huh, I don't remember adding this..... It should add support for `npx create-nx-workspace --version` to print the cached version.... Not bad functionality imho, but I can remove if you want.